blockchain/blockchain.go

```diff
@@ -109,15 +109,13 @@ func (b *Blockchain) Store(block *core.Block, stateUpdate *core.StateUpdate) err
 	})
 }
 
-func (b *Blockchain) VerifyBlock(block *core.Block, stateUpdate *core.StateUpdate) error {
+func (b *Blockchain) IsNextHead(block *core.Block) error {
```

Comment:
I think the previous name still makes sense since `verifyBlock` doesn't imply that we are going to store it.

Reply:
I wanted emphasize that it assumes the block to be sanity checked beforehand. Any suggestions that would have that meaning?

Reply:
I cannot think of a better name than `VerifyBlock`. Usually `Is...()` return bool and we are not doing that here. I think keeping it as verify block with a comment to say what it does should be sufficient.

Reply:
got it, reverting